eternalcore/src/main/java/com/eternalcode/core/teleport/TeleportTask.java

```diff
@@ -46,7 +47,7 @@ public void run() {
                 continue;
             }
 
-            player.teleport(location);
+            PaperLib.teleportAsync(player, location);
```

Comment:
to nie może być async w tasku

Reply:
Fakt, przy zmianie na PaperLiba pomyliłem się :smile: